docs/machine-learning/how-to-guides/serve-model-web-api-ml-net.md

```diff
@@ -0,0 +1,121 @@
+
+[!code-csharp[AddUsings](../../../samples/machine-learning/how-tos/SentimentAnalysisWebAPI/Services/PredictionService.cs#1 "Add necessary usings")]
+
+Remove the existing class definition and add the following code to the PredictionService.cs file:
+
+[!code-csharp[DefinePredictionService](../../../samples/machine-learning/how-tos/SentimentAnalysisWebAPI/Services/PredictionService.cs#2 "Define Prediction Service")]
+
+## Register Predictions Service for Use in Application
+
+While it is possible to create the service every time it needs to be used, when it comes to best practices there are better approaches to that can be taken. One such approach is dependency injection which is built into ASP.NET Core.
+
+The following link provides more information if you want to learn about [dependency injection](https://docs.microsoft.com/aspnet/core/fundamentals/dependency-injection?view=aspnetcore-2.1).
+
+1. Open the Startup.cs class and add the following using statement to the top of the file: 
+
+[!code-csharp[AddUsings](../../../samples/machine-learning/how-tos/SentimentAnalysisWebAPI/Startup.cs#1 "Add necessary usings")]
+
+1. Add the following lines of code to the ConfigureServices method:
+
+[!code-csharp[RegisterServices](../../../samples/machine-learning/how-tos/SentimentAnalysisWebAPI/Startup.cs#2 "Register Services")]
+
+At a high level, this code initializes the objects and services automatically when requested by the application instead of having to manually do it.
+
+## Create Predict Controller
+
+To process our incoming HTTP requests, we need to create a controller.
+
+1. In Solution Explorer, right-click the *Controllers* directory, and then select Add > Controller.
+1. In the Add New Item dialog box, select API Controller Empty and select Add.
+1. In the prompt change the Controller Name field to PredictController.cs. Then, select the Add button. The PredictController.cs file opens in the code editor. Add the following using statement to the top of PredictController.cs:
+
+[!code-csharp[AddUsings](../../../samples/machine-learning/how-tos/SentimentAnalysisWebAPI/Controllers/PredictController.cs#1 "Add necessary usings")]
+
+Remove the existing class definition and add the following code to the PredictController.cs file:
+
+[!code-csharp[DefinePredictController](../../../samples/machine-learning/how-tos/SentimentAnalysisWebAPI/Controllers/PredictController.cs#2 "Define Predict Controller")]
+
+All this is doing is assigning the Prediction service by passing it to the controller's constructor which we get via dependency injection. Then, in the POST method of this controller the Prediction service is being used to make predictions and return the results back to the user if successful.
+
+## Test Web API Locally
+
+1. Once everything is set up, it's time to test the application. To do so, run the application.
```

Comment:
I'd move "Once everything is set up, it's time to test the application." above the numbered list, and simply put "Run the application."